### PR TITLE
PKG-3260

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- new 2023.7.1

https://github.com/python-jsonschema/jsonschema-specifications/tree/v2023.07.1

### CODE REVIEW

File: jsonschema-specifications/meta.yaml
Comment (JeanChristopheMorinPerso) [inline jsonschema-specifications/meta.yaml:1]: I'm guessing this was some kind of testing?

File: jsonschema-specifications/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jsonschema_specifications-{{ version }}.tar.gz
+  sha256: c91a50404e88a1f6ba40636778e2ee08f6e24c5613fe4c53ac24578a5a7f72bb
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (M-Waszkiewicz-Anaconda) [inline jsonschema-specifications/meta.yaml]: -no-deps --no-build-isolation ?

File: jsonschema-specifications/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
```
Comment (M-Waszkiewicz-Anaconda) [inline jsonschema-specifications/meta.yaml]:
```suggestion
    - python
```

File: jsonschema-specifications/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: The JSON Schema meta-schemas and vocabularies, exposed as a Registry
+  dev_url: https://github.com/python-jsonschema/jsonschema-specifications
+  license: MIT
+  license_file: COPYING
```
Comment (M-Waszkiewicz-Anaconda) [inline jsonschema-specifications/meta.yaml]: missing license_family, description

File: jsonschema-specifications/meta.yaml
Comment (cbouss) [reply jsonschema-specifications/meta.yaml:1]: yes, I will remove this file

File: jsonschema-specifications/meta.yaml (hunk 1 above)
Comment (cbouss) [reply jsonschema-specifications/meta.yaml]: sorry, a file I shouldn't have committed.

**Review by skupr-anaconda — APPROVED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**